library/Fission/Environment.hs

```diff
@@ -55,5 +55,5 @@ withEnv key fallback transform = pure (maybe fallback transform) <*> lookupEnv k
 --
 -- >>> getFlag ""
 -- False
```

Comment:
Nonzero chance that this doctest will now fail

Reply:
```shell
 ~/D/W/F/web-api  ╍╍ envy  stack test fission-web-api:fission-doctest

# ...

/Users/expede/Documents/Work/FISSION/web-api/.doctest-tmp/library/Fission/Environment.hs:57: failure in expression `getFlag ""'
expected: False
 but got: Nothing

Examples: 19  Tried: 19  Errors: 0  Failures: 1
```

Reply:
yuup